gadgetchains/PopPHP/FD/1/chain.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Does not work against any versions, and version 4.7.0 does not seem to exist? Last being [4.3.4](https://github.com/popphp/popphp/releases/tag/4.3.4)

Reply:
sorry about the confusion, I used the framework https://github.com/popphp/popphp-framework